src/types/label.js

```diff
@@ -84,24 +84,12 @@ LabelAnnotation.defaults = {
     style: undefined,
     weight: undefined
   },
-  height: undefined,
```

Comment:
Why did you remove `height` and `width`?

Reply:
Aren't those used in `resolveElementProperties` only?

Reply:
> Aren't those used in `resolveElementProperties` only?

Well, they are, to calculate the size of the label when an image or canvas is used, but together many others (i.e. font, padding), when the content is a text. Do we have to remove all options which are affecting the annotation dimension?

Reply:
We don't have to remove any of them, but we can remove the ones that do not have a value.